templates/field/field--node--uid.html.twig

```diff
@@ -0,0 +1,3 @@
+{% for item in items %}
```

Comment:
I'm not sure what this template is for. There is a field_author that should have author names for the byline.

Reply:
I didn't realize there was a field. I've switched everything over to use that.